command_line_assistant/utils/dbus.py

```diff
@@ -0,0 +1,63 @@
+"""Utility module that holds dbus classes."""
+
+from typing import Optional, cast
+
+from command_line_assistant.dbus.constants import (
+    CHAT_IDENTIFIER,
+    HISTORY_IDENTIFIER,
+    USER_IDENTIFIER,
+)
+from command_line_assistant.dbus.interfaces.chat import ChatInterface
+from command_line_assistant.dbus.interfaces.history import HistoryInterface
+from command_line_assistant.dbus.interfaces.user import UserInterface
+
+
+class DbusUtils:
```

Comment:
Like RendererUtils, this is less of a utility and more core functionality. I
would suggest we name this something like `DBusClient`. This is the primary
entrypoint the `c` program uses to access remote objects on the bus.